src/Nest/Resolvers/Writers/TypeMappingWriter.cs

```diff
@@ -274,8 +274,14 @@ private string GetElasticSearchTypeFromFieldType(FieldType? fieldType)
 				switch (propertyType.Name)
 				{
 					case "Int32":
+					case "UInt32":
+					case "Int16":
+					case "UInt16":
+					case "Byte":
+					case "SByte":

```

Comment:
Agreed.  Just pushed another commit that adds support for `byte` and `short.  Also did some minor cleanup of `TypeMappingWriter`.